qml/pages/VenueMapPage.qml

```diff
@@ -48,7 +48,7 @@ BVApp.Page {
 
             onClicked: {
                 var query = venueCoordinate.latitude + "," + venueCoordinate.longitude
-                if (BVApp.Theme.isIos || BVApp.Theme.isOSX) {
+                if (BVApp.Platform.isIos || BVApp.Platform.isMacOS) {
                     Qt.openUrlExternally("https://maps.apple.com/?q=" + query)
```

Comment:
We could even think about just using the Apple link for all cases as apple redirects to Google for other platforms, just try on GNU/Linux  ;)